.github/workflows/ruby-spec.yml

```diff
@@ -0,0 +1,30 @@
+name: ruby-rspec
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

Comment:
So obviously this won't work until we switch over to the `main` branch as the default, but I wanted to get a sense of the publishing pipeline for this gem in case there are other config updates to make prior to making that switch.